### PR TITLE
Lifecycle controller attributes

### PR DESCRIPTION
**Pull request by prajwalpatil25**
terraform-provider-redfish/redfish/provider/resource_redfish_dell_lifecycle_controller_attributes.go (77.6%)

### CODE REVIEW

File: redfish/provider/resource_redfish_dell_lifecycle_controller_attributes.go [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+func (*dellLCAttributesResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
+	tflog.Trace(ctx, "resource_DellLCAttributes delete: started")
+	// Get State Data
+	var state models.DellLCAttributes
+	diags := req.State.Get(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
```
Comment (rounak-adhikary) [inline redfish/provider/resource_redfish_dell_lifecycle_controller_attributes.go:190]: Looks like redundant, please remove if not required.

File: redfish/provider/resource_redfish_dell_lifecycle_controller_attributes.go [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+			}
+		}
+	}
+	d.Attributes = types.MapValueMust(types.StringType, readAttributes)
+	return diags
+}
+
+func getLCAttributes(attributes []*dell.Attributes) (*dell.Attributes, error) {
+	for _, a := range attributes {
+		log.Printf("attributes: %s", a.ID)
```
Comment (rounak-adhikary) [inline redfish/provider/resource_redfish_dell_lifecycle_controller_attributes.go:369]: pls use tflog or remove

File: redfish/provider/resource_redfish_dell_lifecycle_controller_attributes_test.go [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+				ResourceName:  "redfish_dell_lc_attributes.lc",
+				ImportState:   true,
+				ImportStateId: "{\"username\":\"" + creds.Username + "\",\"password\":\"" + creds.Password + "\",\"endpoint\":\"https://" + creds.Endpoint + "\",\"ssl_insecure\":true}",
+				ExpectError:   nil,
+			},
+		},
+	})
+}
+
+func TestAccRedfishLCAttributeImportByFilter(t *testing.T) {
```
Comment (rounak-adhikary) [inline redfish/provider/resource_redfish_dell_lifecycle_controller_attributes_test.go:58]: please add this testcase as 2nd step in TestAccRedfishLCAttributesBasic

File: redfish/provider/resource_redfish_dell_lifecycle_controller_attributes.go (hunk 1 above)
Comment (prajwalpatil25) [reply redfish/provider/resource_redfish_dell_lifecycle_controller_attributes.go]: removed

File: redfish/provider/resource_redfish_dell_lifecycle_controller_attributes_test.go (hunk 3 above)
Comment (prajwalpatil25) [reply redfish/provider/resource_redfish_dell_lifecycle_controller_attributes_test.go]: done

File: redfish/provider/resource_redfish_dell_lifecycle_controller_attributes.go (hunk 2 above)
Comment (prajwalpatil25) [reply redfish/provider/resource_redfish_dell_lifecycle_controller_attributes.go]: removed

**Review by rounak-adhikary — APPROVED** [review]

**Review by Sakshi-dell — APPROVED** [review]

**Pull request merged**